src/common/loggers/mat_logger.h

```diff
@@ -16,25 +15,23 @@
 
 namespace CsvLog {
 
-constexpr size_t kFrames = 1000;
+constexpr size_t kFrames = 10;
+constexpr size_t kFrameStart = 100;
 constexpr size_t kSCs = 304;
 constexpr size_t kBSAnts = 8;
```

Comment:
These look suspecious..... Why are these fixed values, should you be using the configurations?